src/components/learner-credit-management/AssignmentDetailsTableCell.jsx

```diff
@@ -3,11 +3,20 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { Stack, Hyperlink } from '@edx/paragon';
 
+import { sendEnterpriseTrackEvent } from '@edx/frontend-enterprise-utils';
 import { configuration } from '../../config';
 import EmailAddressTableCell from './EmailAddressTableCell';
+import EVENT_NAMES from '../../eventTracking';
 
-const AssignmentDetailsTableCell = ({ row, enterpriseSlug }) => {
+const AssignmentDetailsTableCell = ({ row, enterpriseSlug, enterpriseId }) => {
   const { ENTERPRISE_LEARNER_PORTAL_URL } = configuration;
+  const handleOnClick = () => sendEnterpriseTrackEvent(
+    enterpriseId,
+    EVENT_NAMES.LEARNER_CREDIT_MANAGEMENT.BUDGET_DETAILS_ASSIGNED_DATATABLE_VIEW_COURSE,
+    {
+      courseUUID: row.original.uuid,
```

Comment:
Is the `row.original.uuid` referring to the content assignment UUID or the course UUID? I believe it's referring to the content assignment UUID. I believe you might want to log the content key here?

Reply:
resolved by referencing the contentKey 👍🏽